src/sentry/models/dashboard_widget.py

```diff
@@ -79,6 +89,26 @@ class Meta:
     __repr__ = sane_repr("widget", "type", "name")
 
 
+class DashboardWidgetMetricsQuery(DashboardWidgetQueryBase):
+    """
+    A metrics query in a dashboard widget.
+    """
+
+    __core__ = True
+
+    #: A list of metrics tag names to group by
+    groupby = ArrayField()
+    #: every metric needs a project
+    project = FlexibleForeignKey("sentry.Project")
+
+    class Meta:
+        app_label = "sentry"
+        db_table = "sentry_dashboardwidgetmetricsquery"
+        unique_together = (("widget", "order"),)
+
+    __repr__ = sane_repr("widget", "name")
+
+
```

Comment:
Do we necessarily need to have two tables? They look to have mostly the same field, could we just null or leave blank the fields that aren't used by the other widget type? Or possibly some sort of enum field to make filtering easier instead?